src/modality-specific-files/microelectrode-electrophysiology.md

```diff
@@ -0,0 +1,733 @@
+We recommend using the following keys to describe the setup:
+
+### Institution Information
+
+{{ MACROS___make_sidecar_table("microephys.microephysInstitutionInformation") }}
+
+### Setup Information
+
+{{ MACROS___make_sidecar_table("microephys.microephysSetup") }}
+
+### Processing Information
+
+{{ MACROS___make_sidecar_table("microephys.microephysProcessing") }}
+
+### Additional Procedure Information
+
+Furthermore, additional information can be stored about the recording procedure.
+We RECOMMEND to use a dedicated `Procedure` node with the following keys:
+
+-   `Pharmaceuticals`
+-   `Sample`
+-   `Supplementary`
+
+<!-- TODO: Yarik replaced Pharmaceuticals with PharmaceuticalName and others for now but we might look to define list of Pharmaceuticals of records with PharmaceuticalDoseAmount and PharmaceuticalDoseUnit -->
+
+#### Pharmaceuticals
+
+For each pharmaceutical we RECOMMEND to use a dedicated node with the name of the Pharmaceuticals containing the following administration details:
+
+{{ MACROS___make_sidecar_table("microephys.microephysPharmaceuticals") }}
+
+#### Sample
+
+{{ MACROS___make_sidecar_table("microephys.microephysSample") }}
+
+#### Supplementary
+
+{{ MACROS___make_sidecar_table("microephys.microephysSupplementary") }}
+
+### Task Information
+
+If the OPTIONAL [` task-<label>`](../appendices/entities.md#task) is used, the following metadata SHOULD be used.
+
+{{ MACROS___make_sidecar_table("microephys.microephysTaskInformation") }}
+
+### Example `*_icephys.json` and `*_ecephys.json`
+
+<!-- TODO: below there is Procedure.Pharmaceuticals which is not standardized since ATM there is only single pharmaceutical "allowed" and we have no "Procedure" -->
+
+```JSON
+{
+  "PowerLineFrequency": 50,
+  "Manufacturer": "OpenEphys",
+  "ManufacturerModelName": "OpenEphys Starter Kit",
+  "ManufacturerModelVersion": "OEPS-9031",
+  "SamplingFrequency": 30000,
+  "SamplingFrequencyUnit": "Hz",
+  "Location": "Institut de Neurosciences de la Timone, Faculté de Médecine, 27, boulevard Jean Moulin, 13005 Marseille - France",
+  "Software": "Cerebus",
+  "SoftwareVersion": "1.5.1",
+  "Creator": "John Doe",
+  "Maintainer": "John Doe jr.",
+  "Procedure": {
+    "Pharmaceuticals": {
+      "isoflurane": {
+        "PharmaceuticalName": "isoflurane",
+        "PharmaceuticalDoseAmount": 50,
+        "PharmaceuticalDoseUnit": "ug/kg/min"
+      },
+      "ketamine": {
+        "PharmaceuticalName": "ketamine",
+        "PharmaceuticalDoseAmount": 0.1,
+        "PharmaceuticalDoseUnit": "ug/kg/min"
+      }
+    },
+    "Sample": {
+      "BodyPart": "BRAIN",
+      "BodyPartDetails": "Motor Cortex"
+    },
+    "ProbeContours": {
+      "p021": {
+        "Contour": [
+          [0, 0, 0],
+          [0, 10, 0],
+          [1, 11, 0],
+          [2, 10, 0],
+          [2, 0, 0]
+        ],
+        "Unit": "mm"
+      }
+    },
```

Comment:
```suggestion
```

I think we agreed the solution here was to either:
1. Reference a probe in the probe interface library or
2. Define the custom probe interface file.

We had considered putting the information here, but decided against it.